YOUNGSUCIOGUAPO/Ecommerce-Frontend
# Feature/cart

#### What does this PR achieve?
Implemented the cart page and it features, like add to cart, decrease and increase cart quantity and remove item functionality.
 
#### Any background context?
user clicked on the add to cart button from the homepage, items are added in the cart, and the user can view items in the cart, and as well as in crease or decrease the quantity on the cart page. 

#### Manual Testing Steps
- Pull the code to your local machine
- Test for the stated changes
- click on add to cart to add product to the cart
- navigate to the cart page to  see the products in the cart

#### Screenshots
 <img width="1440" alt="Screenshot 2024-09-10 at 10 22 14" src="https://github.com/user-attachments/assets/571c0c25-ed78-4087-97eb-59e650d91090">
<img width="1440" alt="Screenshot 2024-09-10 at 10 12 08" src="https://github.com/user-attachments/assets/bf16de0e-2547-45a8-b7ff-16370e5c8dce">
#### Relevant Azure DevOps Ticket
#
